index.html

```diff
@@ -2229,6 +2229,7 @@ <h4>About</h4>
         <!-- -------------------------------------------------- -->
       </div>
       <hr />
+      <a href="#" id="toTopBtn" class="cd-top text-replace js-cd-top cd-top--is-visible cd-top--fade-out" data-abc="true"></a>
```

Comment:
please remove all the `class` and the `data`attributes, they are not being used for anything. You need to keep only `href` and `id`